dallinger/frontend/static/scripts/dallinger2.test.js

```diff
@@ -101,7 +101,7 @@ describe('identity', function () {
 
   beforeEach(function () {
     window.history.pushState({}, 'Test Title',
-      'test.html?recruiter=hotair&hitId=HHH&assignmentId=AAA&workerId=WWW&mode=debug'
+      'test.html?recruiter=hotair&hitId=HHH&assignmentId=AAA&workerId=WWW&uniqueId=WWW:AAA&mode=debug'
```

Comment:
Are you sure `uniqueId` is meant to be in this URL?

Reply:
You are right, `uniqueId`  shouldn't be in the URL. Removed.